packages/server/graphql/types/GitHubIntegration.ts

```diff
@@ -69,64 +68,45 @@ const GitHubIntegration = new GraphQLObjectType<any, GQLContext>({
     issues: {
       type: new GraphQLNonNull(GitHubIssueConnection),
       description:
-        'A list of issues coming straight from the jira integration for a specific team member',
+        'A list of issues coming straight from the GitHub integration for a specific team member',
       args: {
         first: {
-          type: GraphQLInt,
-          defaultValue: 100
+          type: GraphQLInt
         },
         after: {
-          type: GraphQLISO8601Type,
-          description: 'the datetime cursor'
+          type: GraphQLString,
+          description: 'a unique cursor id from GitHub'
         },
         queryString: {
           type: GraphQLString,
           description: 'A string of text to search for'
-        },
-        nameWithOwnerFilters: {
-          type: GraphQLList(GraphQLNonNull(GraphQLID)),
-          descrption: 'A list of repos to restrict the search to'
         }
       },
-      resolve: async (
-        {teamId, userId, accessToken, cloudIds},
-        {first, queryString, projectKeyFilters},
-        context
-      ) => {
-        const {authToken} = context
+      resolve: async ({teamId, userId, accessToken}, {first, queryString, after}, {authToken}) => {
         const viewerId = getUserId(authToken)
-        if (viewerId !== userId) {
-          const err = new Error('Cannot access another team members issues')
-          standardError(err, {tags: {teamId, userId}, userId: viewerId})
-          return connectionFromTasks([], 0, err)
-        }
-        if (!accessToken) {
-          const err = new Error('Not integrated with GitHub')
-          standardError(err, {tags: {teamId, userId}, userId: viewerId})
-          return connectionFromTasks([], 0, err)
+        if (viewerId !== userId || !accessToken) {
+          const error = new Error(
+            viewerId !== userId
+              ? 'Cannot access another team members issues'
+              : 'Not integrated with GitHub'
+          )
+          standardError(error, {tags: {teamId, userId}, userId: viewerId})
+          return {error, edges: [], pageInfo: []}
         }
         const manager = new GitHubServerManager(accessToken)
-        const projectKeyFiltersByCloudId = {}
-        if (projectKeyFilters?.length > 0) {
-          projectKeyFilters.forEach((globalProjectKey) => {
-            const [cloudId, projectKey] = globalProjectKey.split(':')
-            projectKeyFiltersByCloudId[cloudId] = projectKeyFiltersByCloudId[cloudId] || []
-            projectKeyFiltersByCloudId[cloudId].push(projectKey)
-          })
-        } else {
-          cloudIds.forEach((cloudId) => {
-            projectKeyFiltersByCloudId[cloudId] = []
-          })
+        const validQueryStr = queryString.replace(/is:pr/g, '')
```

Comment:
If a user enters `is:pr` and `is:issue`, the nodes returned from the query are empty. This is a problem as the `issueCount` includes the empty nodes

Reply:
this smells like a problem with the inline fragment that you might be using?